x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_indices/indices_logic.test.ts

```diff
@@ -31,7 +33,7 @@ const DEFAULT_VALUES = {
   status: Status.IDLE,
 };
 
-const apiIndex: ViewSearchIndex = {
+const apiIndex: ElasticsearchViewIndex = {
```

Comment:
Maybe these should go to a shared mock file?

Reply:
Oh I did that, but forgot to delete them from this test file 🤦